src/main/java/com/conveyal/gtfs/graphql/GraphQLGtfsSchema.java

```diff
@@ -644,6 +651,12 @@
                     // DataFetchers can either be class instances implementing the interface, or a static function reference
                     .dataFetcher(new JDBCFetcher("patterns"))
                     .build())
+            .field(newFieldDefinition()
+                .name("shapes_as_polylines")
```

Comment:
Nit: would like this named `shapes_as_encoded_polylines`.

Reply:
I think the format of the polylines is fairly obvious by looking at the response?